lib/tasks/enable_features.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+task enable_features: :environment do
+  User.all.each do |user|
```

Comment:
So technically this script isn't enabling this feature for the user you listed, it's enabling the feature for everyone for each user you have in your database.

You could just enable them all without the loop or if you want to enable the feature for just `staff` which we have defined as a group https://github.com/education/classroom/blob/44efed0852076877840d2dcd387389f165e0eaa5/config/initializers/flipper.rb#L21-L24

We could just make a call that enables it for the group. I would check out https://github.com/jnunemaker/flipper/blob/master/examples/enabled_for_actor.rb

Reply:
Great thanks I'll check the documentation !